src/main/cljc/jarl/eval.cljc

```diff
@@ -488,7 +488,10 @@
             (do
               (log/tracef "function <%s> threw error: %s" name (ex-message e))
               (if (true? (get state :strict-builtin-errors))
-                (throw e)
+                (let [[rego-type & m] (str/split (ex-message e) #":")
```

Comment:
Could be moved to a helper function in `jarl.exceptions`, I think. Something like `jarl.exceptions/add-builtin-name`, that takes the exception and builtin name as args?

Reply:
With the above change, this is only one line of code anyway, so might just keep it like it is unless we need to do this in more places later.